src/euphorie/client/browser/risk.py

```diff
@@ -750,28 +763,33 @@ def __call__(self):
         self.title = context.parent.title
 
         # Italian special
+        if self.is_custom_risk:
+            self.risk.description = u""
+            self.risk.evaluation_method = u""
         if self.italy_special:
-            self.skip_evaluation = True
+            if self.risk and (
+                self.risk.type == "top5" or self.risk.evaluation_method == "fixed"
+            ):
+                self.skip_evaluation = True
             measures_full_text = True
         else:
             measures_full_text = False
-        if self.is_custom_risk:
-            self.risk.description = u""
-        else:
+        if not self.is_custom_risk:
             existing_measures = [
                 txt.strip() for (txt, active) in self.get_existing_measures() if active
             ]
             solutions = []
             for solution in self.risk.values():
                 if not ISolution.providedBy(solution):
                     continue
+                description = (
+                    getattr(solution, "description", "") or "").strip()
+                prevention_plan = (
+                    getattr(solution, "prevention_plan", "") or "").strip()
+                match = description
                 if measures_full_text:
-                    match = u"%s: %s" % (
-                        (getattr(solution, "description", "") or "").strip(),
-                        (getattr(solution, "prevention_plan", "") or "").strip(),
-                    )
-                else:
-                    match = solution.description.strip()
+                    if prevention_plan:
```

Comment:
Minor: join the ifs -> `if measures_full_text and prevention_plan:`.
Implies less indentation

Reply:
ok, done